src/components/bottom-sheet/dashboard-menu-bottom-sheet/dashboard-menu-bottom-sheet.tsx

```diff
@@ -161,6 +178,16 @@ export class DashboardMenuBottomSheetComponent extends React.Component<
                             iconName: IconValues.FLASH_OFF,
                             onPress: () => this.connectedWebsites()
                         })}
+                    {Platform.OS !== 'web' &&
```

Comment:
I think we also should add this functionality on web, it would be helpful.
HINT: make sure if you add this on web, it will be visible

Reply:
make sure it's visible and working